lib/react_on_rails/server_rendering_pool.rb

```diff
@@ -11,6 +11,15 @@ def self.reset_pool
       @js_context_pool = ConnectionPool.new(options) { create_js_context }
     end
 
+    def self.reset_pool_if_server_bundle_was_modified
+      return unless ReactOnRails.configuration.reload_server_js_every_request
+      file_mtime = File.mtime(ReactOnRails.configuration.server_bundle_js_file)
+      @server_bundle_timestamp ||= file_mtime
+      return if @server_bundle_timestamp == file_mtime
+      ReactOnRails::ServerRenderingPool.reset_pool
+      @server_bundle_timestamp = file_mtime
```

Comment:
This line should be where we [actually read the file, right](https://github.com/shakacode/react_on_rails/pull/75/files#diff-10c7b74a600ef3ecde9dca86ed42c16eR90)?

So long as we don't have multiple threads, there's no timing issues. Consider if there could be race conditions.